diesel_compile_tests/tests/ui/sql_type_bad_options.stderr

```diff
@@ -36,17 +34,17 @@ warning: Option ary_oid has no effect
 21 | #[postgres(oid = "NaN", ary_oid = "1")]
    |                         ^^^^^^^
 
-error: Missing required option array_oid
+error: Missing required option `array_oid`
   --> $DIR/sql_type_bad_options.rs:21:3
    |
 21 | #[postgres(oid = "NaN", ary_oid = "1")]
    |   ^^^^^^^^
 
-error: Expected a number
-  --> $DIR/sql_type_bad_options.rs:21:18
+error: `postgres` must be in the form `postgres(...)`
```

Comment:
Isn't that wrong ?

Reply:
No, the diff is just a bit confusing because the `expected a number` error appeared twice before